.github/workflows/nix-build.yml

```diff
@@ -0,0 +1,29 @@
+name: nix build
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**/*.rs'
+      - '.github/workflows/nix.yml'
```

Comment:
Sorry to pop up suddenly, but shouldn't this be `.github/workflows/nix-build.yml`?

Reply:
fair point, that might be the reason why they didn't started.